movie-app/src/App.js

```diff
@@ -12,10 +15,11 @@ function App() {
   },[])
 
   return (
-    <div>
+    <div className="App">
```

Comment:
Try to keep css class names kebab-case:  ex: `app-main-container` and `kebab-case`